scripts/zones/Abyssea-Vunkerl/npcs/qm10.lua

```diff
@@ -4,16 +4,17 @@
 -- Spawns Armillaria
 -- !pos -396 -31 196 217
 -----------------------------------
-require("scripts/globals/abyssea")
+require('scripts/globals/abyssea')
+require('scripts/globals/items')
 -----------------------------------
 local entity = {}
-
 entity.onTrade = function(player, npc, trade)
-    -- xi.abyssea.qmOnTrade(player, npc, trade)
+    local ID = zones[player:getZoneID()]
```

Comment:
There doesn't need to be a lookup in these scripts for ID, since we know what zone the player is in.  Instead, this should be in the require block as something like:
`local ID = require('scripts/zones/Abyssea-Vunkerl/IDs')`

Reply:
Please note: This change also impacts every PR you have raised since.  Not going to review the others individually at this time.

Reply:
This was taken directly from the already merged coding in Abyssea - La T and Kon. I suppose when this section of Abyssea gets an audit and we know exactly how you want it all to mesh we can worry about them then.

Reply:
Looking at Abyssea LaTheine it does have same type https://github.com/LandSandBoat/server/blob/base/scripts/zones/Abyssea-La_Theine/npcs/qm_briareus.lua#L17

Reply:
>  I suppose when this section of Abyssea gets an audit and we know exactly how you want it all to mesh we can worry about them then.

previous things not having been updated is not a good reason to add more that needs updated, and that is not your call to make.